Makefile

```diff
@@ -132,6 +132,9 @@ showpath:
 $(BUNDLED_DEPS): src/js/trove/require-node-dependencies.js
 	browserify src/js/trove/require-node-dependencies.js -o $(BUNDLED_DEPS)
 
+# src/arr/trove/ast-visitors.arr: src/arr/trove/ast.arr src/arr/compiler/autogenerate.arr
```

Comment:
Should this only depend on ast.arr once things are merged? Autogenerate will be built into the compiler.